doc/appendix-de-kassensichv/reference-tables/reference-tables.md

```diff
@@ -446,7 +446,7 @@ Each subitem as described in the table below:
 | `BON_TYP` | Receipt type / action type| String (30) | Deducted from `ftReceiptCase` |
 | `BON_NAME` | Additional description related to the `BON_TYP` | String (60) | Mandatory if `BON_TYPE` is "AVSonstige", otherwise optional, can be sent via `ftReceiptCaseData` in JSON format. To send, add the key value pair `ReceiptName ` e.g. `"ftReceiptCaseData":"{ ..., "ReceiptName":"Sonstige Sonderwurst", ... }"` |
 | `TERMINAL_ID` | Mandatory, ID of the terminal that was used to record this receipt | String (50) | `cbTerminalID` |
-| `BON_STORNO` | Cancellation indicator | String | Filled by ft. Mandatory: if the receipt is a reverse/voided receipt, then it must be marked with the `ftReceiptCaseFlag` `0x0000000000040000` and must reference the initial receipt via `cbPreviousReceiptReference`. The referenced initial receipt must be part of the current cashpoint closing.|
+| `BON_STORNO` | Cancellation indicator | String | Filled by ft. Mandatory: if the receipt is a reverse/voided receipt, then it must be marked with the `ftReceiptCaseFlag` `0x0000000000040000` and has to reference the initial receipt via `cbPreviousReceiptReference`. |
```

Comment:
Because the "initial receipt" is the first "Start receipt" of the security mechanism, I would delete and replace the word "initial" from the sentence:
"and has to reference the initial receipt via `cbPreviousReceiptReference`. |" eg to:
"and has to be reference to the related receipt via `cbPreviousReceiptReference`. |"

Description of the Start receipt (initial receipt) 
https://docs.fiskaltrust.cloud/doc/interface-doc/doc/general/cash-register-integration/cash-register-integration.html#zero-receipt

Reply:
Hi Axel, thank you for your suggestion. Sounds good, I will replace it.